pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -142,6 +173,16 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
```

Comment:
Does the order of execution matter at all, any value to deploying the target and service before the pollers?

Reply:
> Does the order of execution matter at all, any value to deploying the target and service before the pollers?

It does not.  The targets don't look at anything.  The pollers use the information about what is live to decide what to sample.